### PR TITLE
Lint rubocop

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

**Comment by ramereth** [comment]
@armiller the tentant environment variable needs to be updated to ``OS_TENANT_NAME``.

**Comment by ramereth** [comment]
@armiller it also needs ``floating_ip_pool: <%= ENV['OS_FLOATING_IP_POOL'] %>``

**Comment by ramereth** [comment]
@armiller you also need to update the server_spec tests to use the newer format :)

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+AllCops:
+  Include:
+    - '**/Berksfile'
+    - '**/Cheffile'
+  Exclude:
+    - 'metadata.rb'
+
+Metrics/AbcSize:
+  Max: 9001
```
Comment (jordane) [inline .rubocop.yml]: It's over 9000!!!

Comment (ramereth) [inline .rubocop.yml]: I was going to ask about that ... why was this triggered?

Comment (armiller) [inline .rubocop.yml]: https://www.youtube.com/watch?v=SiMHTK15Pik

ABC. There was a lot of logic in that one function.

http://c2.com/cgi/wiki?AbcMetric

**Comment by armiller** [comment]
@ramereth we good now?

File: .rubocop.yml (hunk 1 above)
Comment (jordane) [inline .rubocop.yml]: I will rewrite the function in a future PR and reduce this back to default. But @armiller and I agreed to punt for now.

**Comment by jordane** [comment]
I'm +1.

**Comment by armiller** [comment]
One quick note. The opcache kitchen suites are for my other branch I'm working on. Its fine for them to be pulled in.

**Comment by ramereth** [comment]
Probably unrelated, but I'm getting the following error when trying to converge the apc suite:

```* No candidate version available for php-apc```

Might be an easy fix or something you do in another PR but its concerning.

**Comment by armiller** [comment]
@ramereth php-apc is deprecated. I am working on the opcache pr that will deprecate apc.

**Comment by ramereth** [comment]
Alright, :+1:

**Pull request merged**